doctr/models/classification/predictor/pytorch.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (C) 2021, Mindee.
+
+# This program is licensed under the Apache License version 2.
+# See LICENSE or go to <https://www.apache.org/licenses/LICENSE-2.0.txt> for full license details.
+
+from typing import List, Union
+
+import numpy as np
+import torch
+from torch import nn
+
+from doctr.models.preprocessor import PreProcessor
+
+__all__ = ['OrientationClassifier']
```

Comment:
Also, I'm wondering whether we should keep "Predictor" in all our high level objects to make it clear :thinking: 
What do you think?

Reply:
what do you think of `orientation_predictor` ?

Reply:
or maybe `crop_orientation_predictor` to be clearer